### ISSUE TITLE
fontSize changes result in empty space around lines

### ISSUE THREAD

**Issue by phillipCouto** [diegomura/react-pdf#95]
I am testing out react-pdf as an alternative to our current approach and so far it is very interesting. I really like the concept to use react to define the pdf versus individual render calls.

To the point when I try to create lines of text in a PDF changing the fontSize style is resulting in whitespace about and below the text. It almost seems that the logic to compute the size of the element still thinks it is the default 18.

Here is a PDF illustrating what I am seeing: 
[example.pdf](https://github.com/diegomura/react-pdf/files/1091524/example.pdf)

Here is the code used to generate the example:
```js
import React from 'react';
import { Page, Text, View, Document, StyleSheet } from '@react-pdf/core';
import ReactPDF from '@react-pdf/node';

// Create styles
const styles = StyleSheet.create({
  page: {
    flexDirection: 'row',
  },
  textSmall: {
    fontSize:8,
    margin:0,
    padding:0,
    height:10,
  },
  textLarge: {
    fontSize: 12,
    margin:0,
    padding:0,
  }
});

// Create Document Component

const lines = (count) => {
  let lines = [];
  for (let i = 0; i < count;i++) {
    let s = {};
    if (count > 20) s = styles.textSmall;
    else if (count > 10) s = styles.textLarge;
    lines.push(<Text key={i} style={s}>
      {'Line #' + (i+1)}
    </Text>);
  }
  return lines;
};

const MyDocument = () => (
  <Document>
    <Page size="A4">
      <View>
      {lines(40)}
      </View>
    </Page>
    <Page size="A4" style={styles.page}>
      <View>
      {lines(20)}
      </View>
    </Page>
    <Page size="A4" style={styles.page}>
      <View>
      {lines(10)}
      </View>
    </Page>
  </Document>
);

ReactPDF.render(<MyDocument />, `${__dirname}/example.pdf`);
```

**Comment by diegomura** [comment]
Thanks for reporting this @phillipCouto !
I think I'll have some time today to test this. It may be related with what you said

**Comment by diegomura** [comment]
I think this is happening because the npm package is not updated. So sorry about that 😄 . Look how this gets rendered with the updated codebase:

[example.pdf](https://github.com/diegomura/react-pdf/files/1093380/example.pdf)

I'll publish a new version to npm so you can test it. 
I'm very excited about you wanting to use react-pdf in your project, but please remember this is not production-ready. It will soon, but it's sometimes had to find time to work on OSS 😅

**Comment by diegomura** [comment]
I released `v0.2.0`. Please update to it and try running your example again.

**Comment by phillipCouto** [comment]
Thank you @diegomura that did fix it. I understand react-pdf is very green right now and I was just looking at it from a potential project to migrate to once it is in a state of stability. =)
